[docs] Add authentication against private Git repositories (#763)
chaoss/grimoirelab-perceval: README.md

$ perceval git 'https://github.com/chaoss/grimoirelab-perceval.git' --from-date '2016-01-01'
```

> If you need to execute this command against a **private Git repository**, so, you must set the `username` and `password`:

Reviewer: ```suggestion
To run the backend against a **private git repository**, you must pass the credentials directly in the URL:
```

```

> perceval git https://username:password@repository

Reviewer: ```suggestion
$ perceval git https://username:key@repository-url
```

```

For example, for private GitHub repositories:

```

> perceval git https://username:password@github.com/organization/repository.git

Reviewer: ```suggestion
$ perceval git https://username:api_token@github.com/chaoss/grimoirelab-private
```

Author: Done at https://github.com/chaoss/grimoirelab-perceval/pull/763/commits/526afd7d1fded8b702e5cbe752e2bd1160042b8a
